### ISSUE TITLE
ShareText pullquote accessibility improvements: ARIA, HTML, labelling

### ISSUE THREAD

**Issue by fstorr** [hackoregon/civic#950]
## What
The pullquote / `<blockquote>` elements need improving from an accessibility perspective:

1. The buttons don't have an `aria` attribute to describe that they open pop-up content.
2. The citation should be wrapped in a `<cite>` element which is inside a `<footer>` element (see the [reference example in the HTML 5.3 spec](https://www.w3.org/TR/html53/grouping-content.html#the-blockquote-element)).
3. I'm not convinced that having the entire text of the quote as the button's accessible name is the best idea.
4. The Twitter logo doesn't have an `aria-role` to describe it as an image or an `aria-label` to label it.
5. Not accessibility, but [CSS quotes](https://css-tricks.com/almanac/properties/q/quotes/) can be used instead of HTML ones for the text content.
6. Because there's no hover state for the cursor when the quote is moved over, it's not at all obvious that the content is interactive. Adding a suitable cursor would help that.

**Comment by fstorr** [comment]
The `role` and `aria-label` issue is blocked because its not supported in the package. [I've opened an issue](https://github.com/nygardk/react-share/issues/258#issue-491478315).

**Comment by jaronheard** [comment]
The `role` and `aria-label` issue is no longer blocked with the release of `react-share 4.0.0` See https://github.com/nygardk/react-share/issues/258
